llvm/tools/llvm-readobj/ELFDumper.cpp

```diff
@@ -5031,7 +5031,7 @@ template <class ELFT> void GNUELFDumper<ELFT>::printCGProfile() {
   OS << "GNUStyle::printCGProfile not implemented\n";
 }
 
-template <class ELFT> void GNUELFDumper<ELFT>::printBBAddrMaps() {
+template <class ELFT> void GNUELFDumper<ELFT>::printBBAddrMaps(bool) {
```

Comment:
I feel like it's more common practice to comment out unused parameter names, rather than omit them entirely. Especially here, it means that if GNUStyle is ever implemented, it's clear what the option means without having to look it up.